t/HealthCheck.t

```diff
@@ -335,24 +339,28 @@ my $nl = Carp->VERSION >= 1.25 ? ".\n" : "\n";
     is $c->check, {
         'id'      => 'main',
         'status'  => 'CRITICAL',
+        'runbook' => 'https://runbook-main.url',
```

Comment:
Use a reserved TLD (like .test) in tests.

Reply:
Fixed